docs/using/staking.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Staking
+authors: "Tim McMackin"
+last_update:
+  date: 17 December 2024
+---
+
+Staking is the process of temporarily locking tez on the Tezos platform.
+Staked tez stays in the staker's account, but they cannot transfer that tez while it is staked.
+They can unstake the tez and make it spendable after a delay.
+Staking is an important part of keeping Tezos secure and earning rewards from it.
+
+Two main groups stake on Tezos:
+
+- Bakers must stake 6,000 tez to receive the right to bake blocks and to receive rewards for baking those blocks.
+Their staked tez ensures that they bake correctly, because part of their stake is taken ("slashed") if they misbehave.
+For more information about staking for baking purposes, see [Baking](/architecture/bakers).
+
+- Any Tezos user can stake any amount of tez with a baker and earn rewards based on how much they stake.
```

Comment:
de-duplicate "any amount" and "based on how much", which are detailed just after:
```suggestion
- Any Tezos user can stake tez with a baker and earn rewards for them.
```